notebooks/a00_introduction_to_geomstats.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>NIT: in spherical coordinates, see Figure (a) </p><p>^ I would mention "Figure" explicitly, it improves readability.</p><p>Same for the other references to Figures in the notebooks.</p><p> </p><p>NIT: which is computationally more efficient in terms of memory requirements, and less mentally...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>done</p>